lib/pure/asyncmacro.nim

```diff
@@ -35,14 +35,11 @@ template createCb(retFutureSym, iteratorNameSym,
 
         if next == nil:
           if not retFutureSym.finished:
-            let msg = "Async procedure ($1) yielded `nil`, are you await'ing a " &
-                    "`nil` Future?"
+            let msg = "Async procedure ($1) yielded `nil`, are you await'ing a `nil` Future?"
```

Comment:
Our line length is supposed to be 80 chars afaik :)

Reply:
it's not observed strictly though, and breaking comments is one thing but breaking strings is IMO bad, making it harder to grep for strings.

see also https://github.com/timotheecour/Nim/issues/649 which i intend to turn into an RFC